content/v2.0/reference/syntax/annotated-csv.md

```diff
@@ -177,29 +180,81 @@ csvData = "#datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3
+m,cpu,host,time_steal,usage_user,nothing,time
+cpu,cpu1,host1,0,2.7,a,1482669077000000000
+cpu,cpu1,host2,0,2.2,b,1482669087000000000
+```
+
+#### Line protocol and linepart annotations
+| Linepart key         | Description                                                           |
+|:------------         |:-----------                                                           |
+| `measurement`        | column value parsed as the measurement                                |
+| `field` (default)    | column header parsed as field key, column value parsed as field value |
+| `tag`                | column header parsed as tag key, column value parsed as tag value     |
+| `time`               | column value parsed as timestamp                                      |
+| `ignore` or`ignored` | column is not included in line protocol                               |
+
+##### Example linepart annotations
+```
+#linepart measurement,tag,tag,field,field,ignored,time
+m,cpu,host,time_steal,usage_user,nothing,time
+cpu,cpu1,host1,0,2.7,a,1482669077000000000
+cpu,cpu1,host2,0,2.2,b,1482669087000000000
+```
+
+Resulting line protocol:
+
+```
+cpu,cpu=cpu1,host=host1 time_steal=0,usage_user=2.7 1482669077000000000
+cpu,cpu=cpu1,host=host1 time_steal=0,usage_user=2.2 1482669087000000000
+```
+
+##### Example linepart, datatype, and default annotations
+```
+#datatype ,,string,double,boolean,long,unsignedLong,duration,
+#linepart measurement,tag,,,,,,,time
+#default test,annotatedDatatypes,,,,,,
+m,name,s,d,b,l,ul,dur,time
+,,str1,1.0,true,1,1,1ms,1
+,,str2,2.0,false,2,2,2us,2020-01-11T10:10:10Z
+```
 
-| Datatype     | Flux type  | Description                                                                        |
-| :--------    | :--------- | :----------                                                                        |
-| boolean      | bool       | a truth value, one of "true" or "false"                                            |
-| unsignedLong | uint       | an unsigned 64-bit integer                                                         |
-| long         | int        | a signed 64-bit integer                                                            |
-| double       | float      | an IEEE-754 64-bit floating-point number                                           |
-| string       | string     | a UTF-8 encoded string                                                             |
-| base64Binary | bytes      | a base64 encoded sequence of bytes as defined in RFC 4648                          |
-| dateTime     | time       | an instant in time, may be followed with a colon : and a description of the format |
-| duration     | duration   | a length of time represented as an unsigned 64-bit integer number of nanoseconds   |
+Resulting line protocol:
+
+```
+test,name=annotatedDatatypes s="str1",d=1,b=true,l=1i,ul=1u,dur=1000000i 1
+test,name=annotatedDatatypes s="str2",d=2,b=false,l=2i,ul=2u,dur=2000i 1578737410000000000
```

Comment:
would d=1.0, 2.0?

Reply:
is u added after each unsignedLong integer?

Reply:
This is how you specify unsigned integers in line protocol.

Reply:
@kelseiv Not in line protocol. Numeric values with the `i` or `u` suffix are parsed as floats.

```
1 - float
1.0 - float
1i - integer
1u - unsigned integer
```